src/client/containers/Profile/index.js

```diff
@@ -1,5 +1,65 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUserFirebaseToken } from '../../firebase/auth';
+import man from '../../assets/images/portraits/man1.jpg';
+import './index.styles.css';
+import { ProfileCardComponent } from '../../components/ProfileCardComponent/ProfileCardComponent';
+import { YourUploadedCVs } from '../../components/MyProfile/YourUploadedCVs/YourUploadedCVs';
+import TitleCvCard from '../../components/MyProfile/TitleCvCard/TitleCvCard';
 
 export default function Profile() {
-  return <div>This is your private profile</div>;
+  const [cvsList, setCvsList] = useState([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  // Here it needs the endpoint to get users' infor like github or  profile image url
+  useEffect(() => {
+    (async () => {
+      const userFirebaseToken = await getUserFirebaseToken();
+      if (userFirebaseToken) {
+        console.log('firebase', userFirebaseToken);
+      }
+    })();
+  },[]);
+  // here we should get all cvs for the current use
+  // will be passed to <YourUploadedCVs>
+  useEffect(() => {
+    (async () => {
```

Comment:
@pdsorensen I think this is not the target endpoint for this task, we need all cvs regarding to this specific user in database,, but this endpoint ```api/users``` giving us the list of cvs for all users.